<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CHarmPercentDialog</class>
 <widget class="QDialog" name="CHarmPercentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget_a" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>301</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>A相电压谐波百分比</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_u_a_way_a">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>B相电压谐波百分比</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_u_b_way_a">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>C相电压谐波百分比</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_u_c_way_a">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>A相电流谐波百分比</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_i_a_way_a">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>B相电流谐波百分比</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_i_b_way_a">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>C相电流谐波百分比</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_i_c_way_a">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_pre_page">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_next_page">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>340</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return_back">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>340</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_a">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>主路A</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_b" native="true">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>381</width>
     <height>301</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>A相电压谐波百分比</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_u_a_way_b">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>B相电压谐波百分比</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_u_b_way_b">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>C相电压谐波百分比</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_u_c_way_b">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>A相电流谐波百分比</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_i_a_way_b">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>B相电流谐波百分比</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_i_b_way_b">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>C相电流谐波百分比</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_i_c_way_b">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_title_b">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>主路B</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
